src/com/projectkorra/projectkorra/airbending/AirSwipe.java

```diff
@@ -243,8 +243,8 @@ public void run() {
 							}
 						}
 						if (entities.size() < MAX_AFFECTABLE_ENTITIES) {
-
-							GeneralMethods.setVelocity(entity, fDirection.multiply(AirSwipe.this.pushFactor));
+							
+							GeneralMethods.setVelocity(AirSwipe.this,entity, fDirection.multiply(AirSwipe.this.pushFactor));

```

Comment:
We don't need the white-space above the line. Additionally, please change to
`GeneralMethods.setVelocity(this, entity, fDirection.multiply(AirSwipe.this.pushFactor));`